src/js/createCase.js

```diff
@@ -0,0 +1,83 @@
+import Cookies from 'js-cookie';
+import * as Sentry from '@sentry/browser';
+import logger from './jwt/logger';
+const log = logger('createCase.js');
+import { spinUpStore } from './redux-config';
+
+// Lit of products that are bundles
+const BUNDLE_PRODUCTS = [
+    'Red Hat OpenShift Cluster Manager',
+    'Red Hat Insights',
+    'Subscription Watch'
+];
+
+// List of products that are individual apps
+const APP_PRODUCTS = [
+    { id: 'automation-hub', name: 'Ansible Automation Hub' },
+    { id: 'automation-analytics', name: 'Ansible Automation Analytics' },
+    { id: 'migrations', name: 'Red Hat Migration Analytics' },
+    { id: 'cost-management', name: 'Red Hat Cost Management' }
+];
+
+function registerProduct() {
+    const { store } = spinUpStore();
+    const currentBundle = store.getState().chrome.activeTechnology;
+    const currentApp = store.getState().chrome.appId;
+
+    // check to see if the bundle is a product
+    if (BUNDLE_PRODUCTS.find(bundle => bundle === currentBundle)) {
+        return currentBundle;
+    };
+
+    // if not, check to see if the app is a product
+    const product = APP_PRODUCTS.find(app => app.id === currentApp);
+    return product?.name;
+}
+
+export function createSupportCase(userInfo, fields) {
+
+    const currentProduct = registerProduct();
+
+    log('Creating a support case');
+
+    fetch(`https://access.${window.insights.chrome.isProd ? '' : 'qa.'}redhat.com/hydra/rest/se/sessions`, {
+        method: 'POST',
+        headers: {
+            Accept: 'application/json',
+            Authorization: `Bearer ${Cookies.get('cs_jwt')}`,
+            'Content-Type': 'application/json'
```

Comment:
You should also add `credentials: 'same-origin'` so this request works in FF - https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API

Reply:
When I tried this in FF `78.2.0esr` the request did not work due to this issue ^, and the new tab did not open. However, in FF `80.0.1` it did work.